.travis.yml

```diff
@@ -30,14 +28,21 @@ cache:
   directories:
     - env/bin
     - env/lib/python2.7/site-packages
-    - node_modules
     - geckodriver
 install:
  - if [ "${TRAVIS_BRANCH}" = "master" -a "${TRAVIS_PULL_REQUEST}" = "false" ]; then rm -rf env; fi
  - touch requirements.txt package.json
  - make env -j2
 before_script:
  - echo "DATABASE_URL=dbname=gratipay" | tee -a tests/local.env local.env
+
+ # Turn off all cron threads.
+ - echo "UPDATE_CTA_EVERY=0" >> local.env
```

Comment:
Aren't we running with `test.env` overriding `local.env`? If we're already setting these values to `0` in `test.env`, is it required to set them to zero here too?